broker-daemon/block-order-worker/index.js

```diff
@@ -462,7 +468,11 @@ class BlockOrderWorker extends EventEmitter {
     }
 
     const allRecords = await getRecords(this.store, BlockOrder.fromStorage.bind(BlockOrder), queryOptions)
-    const recordsForMarket = allRecords.filter((record) => record.marketName === market)
+    let filteredRecords = allRecords.filter((record) => record.marketName === market)
+
+    if (options.side) {
+      filteredRecords = filteredRecords.filter((r) => r.side === options.side)
```

Comment:
doesn't having a `limit` option conflict with filtering after the fact? You end up limiting before filtering which seems counterintuitive.

Reply:
Yeah, you can either limit records, or filter them, you're not able to do both effectively

Reply:
This is an issue from the previous PR

Reply:
Ive added a ticket to adjust this, since it is counter intuitive for a limit